bin/prettier.js

```diff
@@ -36,13 +36,15 @@ const argv = minimist(process.argv.slice(2), {
   alias: { help: "h", version: "v", "list-different": "l" },
   unknown: param => {
     if (param.startsWith("-")) {
+      // eslint-disable-next-line no-console
```

Comment:
If you have to add 23 comments to disable a lint warning, we should instead remove the lint warning. It doesn't add much value and is way more annoying